_data/projects/log-agent.yml

```
@@ -0,0 +1,8 @@
+name: LogAgent
+desc: LogAgent is java deamon process that periodically sends running log files to end point storage such as AWS S3, GCP GCS etc.
+site: https://github.com/sagar-arora/LogAgent
+tags:
+  -good first issue
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/sagar-arora/LogAgent
```

I can see you've got an issue on the project matching this label, so let's use that here for the listing...

```suggestion
  name: good first issue
  link: https://github.com/sagar-arora/LogAgent/labels/good%20first%20issue
```